pages/providers/vpn.html

```diff
@@ -235,8 +235,42 @@ <h3>Related VPN information</h3>
           <li><a href="https://www.ivpn.net/privacy">IVPN.net will collect your email and IP address after sign up</a><br />Read the <a data-toggle="tooltip" data-placement="top" data-original-title="The IP collected at signup is only used for a few seconds by our fraud module and then discarded, it is not stored. Storing them would significantly increase our own liability and certainly would not be in our interest. You're absolutely welcome to signup using Tor or a VPN.">Email statement</a> from IVPN.</li>
           <li><a href="https://medium.com/@blackVPN/no-logs-6d65d95a3016">blackVPN announced to delete connection logs after disconnection</a></li>
           <li><a href="https://gist.github.com/kennwhite/1f3bc4d889b02b35d8aa">Don't use LT2P IPSec, use other protocols.</a></li>
+          <li>
+            <a href="https://www.top10vpn.com/free-vpn-app-investigation/">Free VPN App Investigation</a>
+            <span class="badge badge-warning" data-toggle="tooltip" title="This site has affiliate based recommendations">
+              <a href="https://blog.privacytools.io/the-trouble-with-vpn-and-privacy-reviews"/>Warning</a>
+            </span>
+          </li>
+          <li>
+            <a href="https://vpnpro.com/blog/hidden-vpn-owners-unveiled-97-vpns-23-companies/">Hidden VPN owners unveiled: 101 VPN products run by just 23 companies</a>
+            <span class="badge badge-warning" data-toggle="tooltip" title="This site has affiliate based recommendations">
+              <a href="https://blog.privacytools.io/the-trouble-with-vpn-and-privacy-reviews"/>Warning</a>
+            </span>
+          </li>
+          <li>
+            <a href="https://vpnpro.com/blog/chinese-company-secretly-behind-popular-apps-seeking-dangerous-permissions/">This Chinese company is secretly behind 24 popular apps seeking dangerous permissions</a>
+            <span class="badge badge-warning" data-toggle="tooltip" title="This site has affiliate based recommendations">
+              <a href="https://blog.privacytools.io/the-trouble-with-vpn-and-privacy-reviews"/>Warning</a>
+            </span>
+          </li>
         </ul>
 
       </div>
     </div>
   </div>
+
+  <hr>
+
+  <div class="container">
+      <div class="row">
+        <div class="col-12">
+          <a href="https://invidio.us/watch?v=WVDQEoe6ZWY" target="_blank">
+            <img
+              src="/assets/img/png/3rd-party/this-video-is-sponsored-by-vpn.png"
+              class="img-fluid float-left mr-3"
+              alt="This Video Is Sponsored By ███ VPN">
+          </a>
+        </div>
```

Comment:
This is very large. Feels like it could be `col-lg-6 col-12` with a "Related Videos" header.

```suggestion
        <div class="col-lg-6 col-12">
        <h3>Related Videos</h3>
          <a href="https://invidio.us/watch?v=WVDQEoe6ZWY" target="_blank">
            <img
              src="/assets/img/png/3rd-party/this-video-is-sponsored-by-vpn.png"
              class="img-fluid float-left mr-3"
              alt="This Video Is Sponsored By ███ VPN">
          </a>
        </div>
```